index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="Top">
+            <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
+            <button type="button"><img class="menu" src="images/menu.png" alt="Menu"></button>           
```

Comment:
- Please nest the #hamburger menu# button within the `<nav>` tag. This allows screen reader users to navigate directly to the navigation landmark.